# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gle-graphics/X11R7.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gle-4.0.9c/config.gcc.vanilla	2005-10-09 15:33:37.000000000 +0200
+++ gle-4.0.9c/config.gcc	2005-10-09 15:33:46.000000000 +0200
@@ -61,7 +61,7 @@
 
 ifdef HAVE_LIBX11
 	EXTRA_DEFS		:= $(EXTRA_DEFS) -D GLEX11
-	EXTRA_LIB_PATHS		:= $(EXTRA_LIB_PATHS) -L/usr/X11R6/lib
+	EXTRA_LIB_PATHS		:= $(EXTRA_LIB_PATHS) -L/usr/X11/lib
 	LIBS			:= $(LIBS) -lX11
 endif
 
